src/app.js

```diff
@@ -300,6 +300,27 @@ app.routeAsync("/groups/:groupName")
   .getAsync(endpoints.static.sendGatsbyEntrypoint)
 ;
 
+app.use("/groups/:groupName/settings",
+  endpoints.groups.setGroup(req => req.params.groupName));
+
+app.routeAsync("/groups/:groupName/settings/members")
+  .getAsync(endpoints.groups.listMembers);
+
+app.routeAsync("/groups/:groupName/settings/roles")
+  .getAsync(endpoints.groups.listRoles);
+
+app.routeAsync("/groups/:groupName/settings/roles/:roleName/members")
+  .getAsync(endpoints.groups.listRoleMembers);
+
+app.routeAsync("/groups/:groupName/settings/roles/:roleName/members/:username")
+  .getAsync(endpoints.groups.getRoleMember)
+  .putAsync(endpoints.groups.putRoleMember)
+  .deleteAsync(endpoints.groups.deleteRoleMember)
+;
+
+app.route("/groups/:groupName/settings/*")
+  .all(() => { throw new NotFound(); });
```

Comment:
This makes sense on the CLI level, but the UI renders a "dataset not found" page.

Suggestion: 3c2d31a6d059981a734d67d269af6fd22cf91cf1

Reply:
Ugh, really dislike our Gatsy setup. Your suggested fix is appropriate, though, thanks!

Reply:
Applied as 8e99844.